src/test/java/me/itzg/testing/DockerRuleTestElasticsearch.java

```diff
@@ -0,0 +1,83 @@
+
+		} catch (Exception e) {
+			e.printStackTrace();
+			fail(e.getMessage());
+		}
+	}
+
+	@Test
+	public void testDocker() throws DockerCertificateException, DockerException, InterruptedException {
```

Comment:
I'm guessing this was here to do some sanity testing of things. I think you can remove it since the rule itself does a pull.